packages/sdk/lib/__tests__/unit.test.ts

```diff
@@ -0,0 +1,629 @@
+/**
```

Comment:
This file is getting big and covers a lot of functionality. I think it might help to break this into multiple test files by feature - `validation.test.ts`, `plugins.test.ts`, `logger.test.ts` etc

Reply:
Done